src/OpenTK.Windowing.GraphicsLibraryFramework/GLFWNative.cs

```diff
@@ -23,8 +24,50 @@ static GLFWNative()
                     return IntPtr.Zero;
                 }
 
-                return LibraryLoadHelper.LoadLibrary("glfw", new Version(3, 3), assembly, path);
+                return LoadLibrary("glfw", new Version(3, 3), assembly, path);
             });
+
+            // FIXME: This should be static
+            /*static*/ IntPtr LoadLibrary(string libraryName, Version version, Assembly assembly, DllImportSearchPath? searchPath)
```

Comment:
Also I tried updating the `LangVersion` to 8 so that I could use c# 8 functions but for some reason this wont compile...?